### PR TITLE
Pop up Menu

### PR DESCRIPTION
**Pull request by ehmaddd**
It shows the details of the project on which the user clicks on a pop-up Modal. It takes values from array to populate its structure.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,31 +1,109 @@
...
+  document.querySelectorAll('.work-sample h4')[a].innerHTML = projects[a].title;
+  worksUl = document.querySelectorAll('.work-sample ul');
+
+  for (let b = 0; b < projects[a].languages.length; b += 1) {
+    const li = document.createElement('li');
+    li.appendChild(document.createTextNode(projects[a].languages[b]));
+    li.setAttribute('id', 'element4');
+    worksUl[a].appendChild(li);
+  }
+  // document.querySelectorAll('.work-sample a')[a].href = "project-"+parseInt(a+1);
```
Comment (youmari) [inline script.js]: - [x] it would be better to remove the commented code in order to keep the file clean and pro

File: script.js [hunk 2]
```diff
@@ -1,31 +1,109 @@
...
+
+    for (let b = 0; b < projects[a].languages.length; b += 1) {
+      const li = document.createElement('li');
+      li.appendChild(document.createTextNode(projects[a].languages[b]));
+      // li.setAttribute("id", "element4");
+      modalUl.appendChild(li);
+    }
+
+    document.querySelector('#see-live').href = projects[a].liveVersion;
+    document.querySelector('#see-source').href = projects[a].linkSource;
```
Comment (youmari) [inline script.js:96]: - [x]  When I hover over these buttons unexpected effects happen as in the picture below so please make sure to add the effects on the Figma in order to match the Figma design 
- Yours 
Screenshot 2023-03-08 at 13 01 48
- Figma 
Screenshot 2023-03-08 at 13 07 06

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @ehmaddd , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Great commit messages
✔️ PRO PR
✔️ No linter error 
✔️ Clean Design 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (ehmaddd) [reply script.js]: As per instructions, this line is deleted. Now js code is free of it

File: script.js (hunk 2 above)
Comment (ehmaddd) [reply script.js:96]: According to the instructions and color, hover, pressed guideline by figma template the code is corrected as desired.
Regards

**Review by IngDavidTM — APPROVED** [review]
Hi @ehmaddd  🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

- [x] While you did a wonderful job styling your project :+1:, as a project requirement you should match the design as much as possible to the Figma template. Kindly make the popup not use a scroll bar and instead make it fits the screen size :+1:.

**Your design**

https://user-images.githubusercontent.com/105117832/223717487-b6e13a12-5199-4d35-9b3e-ab8a361c0235.mp4

**Figma**

Screenshot from 2023-03-08 07-51-17


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**